src/charm.py

```diff
@@ -127,6 +128,19 @@ def _get_external_hostname(self) -> str:
             self.config["external_hostname"] if self.config["external_hostname"] else self.app.name
         )
 
+    def _is_setup_completed(self) -> bool:
```

Comment:
It is my understanding that this will create the file in the `charm`container. I think it'd be better to leverage pebble to push and pull a file to/from the discourse container, to guarantee a container restart triggers the generation of static assets.

Reply:
Good point, I updated the code to put the file in discourse container instead.